src/precomputed_quantities.jl

```diff
@@ -15,10 +15,14 @@ end
 function precomputed_quantities!(Y, p, t, colidx)
     ᶜuₕ = Y.c.uₕ
     ᶠw = Y.f.w
-    (; ᶜuvw, ᶜK, ᶜts, ᶜp, params) = p
-    (; ᶜinterp) = p.operators
+    (; ᶠuvw, ᶜuvw, ᶜK, ᶜts, ᶜp, params) = p
+    (; ᶠwinterp, ᶜinterp) = p.operators
+    ᶜJ = Fields.local_geometry_field(axes(Y.c.ρ)).J
     C123 = Geometry.Covariant123Vector
-    @. ᶜuvw[colidx] = C123(ᶜuₕ[colidx]) + C123(ᶜinterp(ᶠw[colidx]))
+    @. ᶜuvw[colidx] = C123(ᶜuₕ[colidx]) + C123(ᶜinterp(ᶠw[colidx])) # u̅
+    @. ᶠuvw[colidx] =
```

Comment:
can we rename these? perhaps `u_tilde` and `u_bar`?

Reply:
Noted, I can make those changes - regarding the vector interpolation, is it useful to define the vector interpolation (contravariant components) operation in `ClimaCore,jl` itself? Or is it more appropriate to define the `Contravariant123Vector(...)` in `ClimaAtmos` in the tendencies?